src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -39,6 +60,11 @@ class AmplitudeExperimentService : Disposable {
         variants.forEach { (key, variant) ->
             storage[key] = variant
         }
```

Comment:
storage.putAll(variants)

Reply:
It's not a part of this PR (no diff for the code).